### ISSUE TITLE
slowrot: the p_max=1 Cauchy-Schwarz rung needs a configuration where the delay expansion converges

### ISSUE THREAD

**Issue by oshaughnessy-junior** [oshaughnessy-junior/research-projects-RIT#159]
Surfaced by the `jax-ile-check` CI gate (#151) on its first run against a `rift_O4d` that includes
the harmonic-widening fix (#142/#143). **Not a likelihood bug** — both evaluators agree — but the
`p_max=1` rung of the Cauchy-Schwarz ladder is not currently a well-posed test, and two of its four
checks have been scoped to `p_max=0` as a result.

## What happens

`test/jax/test_jax_slowrot_cauchy_schwarz.py` builds its data as the exact model implied by the
bank, then checks (A) that rotation matters, (B) the bound, (C) agreement with an explicit
time-domain model, (D) agreement with the numpy NoLoop. At `p_max=1` on the widened bank
(14 bands, `0.5<d|d> = 50988.932339`):

```
(A) static deficit             0.3907 nats        (threshold 1.0)
(B) bound overshoot           -4.108e-03 nats     (VIOLATED)
(C) vs explicit model          3.091e-02 nats  =  6.06e-07 relative   PASSES
(D) vs numpy NoLoop            2.488e-09 nats                          PASSES
```

**The bound overshoot is not the evaluator's.** The numpy NoLoop overshoots by the *same*
`-4.108e-03`, and the two agree with each other to `2.5e-09`:

```
p_max=1   JAX    max lnL = 50988.936447   deficit = -4.108112e-03
          NoLoop max lnL = 50988.936447   deficit = -4.108110e-03
          |JAX - NoLoop| = 2.488e-09
```

So the overshoot lives in the data/reference construction. (C) measures that construction's own
conditioning at `6.06e-07` relative — about **0.03 nats** — so the data carries roughly 8x more
error than the 0.004 nats the bound check is trying to resolve. A bound check cannot be meaningful
under those conditions.

## Why the reference is ill-conditioned here

`INFL=1350` with `fmax=1700` puts the delay expansion far past convergence: the drift over the
segment is `delta_tau ~ Omega * T * |B| ~ 8e-3 s`, so at the top of the band
`2*pi*f*delta_tau ~ 85 >> 1`. The `p`-expansion in `delta_tau` is then divergent at high
frequency, and the explicit reference sum inherits that.

At `p_max=0` none of this applies — the antenna expansion is exact at any `Omega` — and the rung is
exact: bound deficit `+0.000000`, (C) `1.28e-15`.

## Note on the pre-#142 numbers

Before widening, this rung read `(A) 36.4 nats` and `(B) +5.076e-04` (bound respected), which looks
healthier. It was not: the bank was missing its `|n| = 3` bands, so those numbers were measured
against the wrong signal. #142 corrected the model and the rung's real conditioning became visible.
Do not treat the old numbers as a regression target.

## What was done, and what is wanted

Done in #151: (A) and (B) are asserted at `p_max=0` only, with the measurements above recorded
inline and an explicit "do not fix this by widening TOL_BOUND". (C) and (D) still run at `p_max=1`
and are what pin the evaluator there — (D) at `1e-8` absolute on `lnL ~ 5e4` is a tight parity
check, and (C)'s relative arm is justified by the NoLoop showing the identical residual.

Wanted: a `p_max>=1` configuration where the expansion converges, so (A) and (B) can be restored at
`p_max=1`. There is a real tension to resolve — the bound check wants `2*pi*f*delta_tau << 1`
(small `Omega`, small `fmax`) while (A) wants `Omega*T` large enough that rotation matters — so this
needs measuring rather than guessing. Lowering `fmax` for this rung looks more promising than
lowering `INFL`, since it relaxes the conditioning without weakening the rotation signal.

Acceptance: (A) and (B) asserted at `p_max=0` and `p_max=1`, with (C)'s relative residual back near
machine precision rather than `6e-07`, and the chosen configuration's numbers recorded in the test.

**Comment by oshaughnessy-junior** [comment]
Addressed in #163 (draft). Reporting the measurements, because the diagnosis in this issue
turned out to be wrong and the sweep is the reason.

## The `fmax` hypothesis is refuted

The issue proposes lowering `fmax` for this rung, on the grounds that
`2*pi*f*delta_tau ~ 85` puts the delay expansion past convergence. Measured at `INFL=1350`,
`p_max=1`, on the widened bank — `max|2*pi*f*delta_tau|` here is 30.4 by the exact
`max_u |Re delta_tau(u)|`, not 85:

| fmax | 1700 | 1024 | 512 | 256 | 128 | 64 |
|---|---|---|---|---|---|---|
| `max\|2 pi f dtau\|` | 30.4 | 18.3 | 9.17 | 4.58 | 2.29 | 1.15 |
| (A) static deficit | 0.3907 | 0.3838 | 0.3760 | 0.3700 | 0.3642 | 0.3523 |
| (B) overshoot [nats] | 4.11e-3 | 5.72e-3 | 6.56e-3 | 7.06e-3 | 7.27e-3 | 6.86e-3 |
| (C) relative | 6.06e-7 | 5.56e-7 | 5.35e-7 | 4.88e-7 | 5.37e-7 | 1.18e-6 |

A 26x reduction in the convergence parameter, and (C) does not move. The `INFL` axis
(fmax=1700) does move things, but in a way that rules out a configuration fix on its own:

| INFL | 135 | 337.5 | 675 | 1350 | 2700 | 5400 | 10800 |
|---|---|---|---|---|---|---|---|
| (A) p_max=1 | 0.0046 | 0.0281 | 0.1072 | 0.3907 | 1.3037 | 3.8951 | 16.55 |
| (B) overshoot | 5.88e-4 | 1.48e-3 | 2.83e-3 | 4.11e-3 | 1.68e-3 | -2.75e-2 | -0.450 |
| (C) relative | 3.39e-8 | 9.48e-8 | 2.26e-7 | 6.06e-7 | 1.54e-6 | 2.05e-6 | 1.37e-5 |

(A) scales as `Omega^2`, the error as roughly `Omega`. Driving `Omega` down to make (B)
resolvable at `TOL_BOUND = 1e-6` needs `INFL ~ 1`, where (A) is ~1e-7 nats — vacuous. There
is no window along either knob.

## What it actually was

Two defects, localised at `INFL=1350`, `fmax=512`, `p_max=1`, **at zero arrival offset**
(`k=0`, post-phase = identity — so not the circular wrap either):

| check | result |
|---|---|
| data term `<d\|h>` vs the explicit reference | 1.0e-15 — exact |
| model norm `<h\|h>` vs the explicit reference | **1.46e-07** |
| evaluator vs the bank's own U/V contracted by hand | 1.4e-16 |
| bank U/V vs independently built cross terms | 4.3e-16 |
| `hlms_conj` vs the exact conjugate spectrum | 1e-15 |
| the (C) reference vs an independent rebuild | 1.4e-16 |
| swap **V only** for an independent family | −1.4925e-02 → −1.3e-10 |

1. **The Nyquist bin of `time_derivative_weight`.** The two-sided packing carries `+fNyq` but
   not `-fNyq`, so an odd derivative weight has no consistent value there, and
   `conj(h^(p))` vs `(conj h)^(p)` differ by a sign in that one bin. `U` takes both factors
   from the same family and never notices; `V = <chi_a^*|chi_a'>` pairs the two orders and
   does. The sidereal modulation is a sub-bin shift applied as a time-domain phase, so it
   spreads that bin across the whole band — being above `fMax` did not protect it.
   `|H(+fNyq)|` is 0.02–0.14 of `|H(100 Hz)|` for these modes. This is a **library** defect;
   the numpy NoLoop carried it identically, which is why the two agreed to 2.5e-09 while both
   overshot.

2. **The test's reference shift convention.** The bank shifts the *modulated* template
   circularly and repairs the phase with `rotation_post_phase`; the reference modulated on
   the unrolled grid. They differ by `e^{i n Omega T_seg}` on the wrapping samples. `hY^(0)`
   is machine zero there (1.2e-16 of peak), `hY^(1)` is not (5.9e-04) — hence p_max=0 was
   clean.

## Outcome

(A) and (B) are asserted at `p_max=0` **and** `p_max=1`, with no tolerance widened:

|  | p_max=0 | p_max=1 |
|---|---|---|
| (A) static deficit | 4.9865 nats | 3.9234 nats |
| (B) bound deficit | +0.000000e+00 | +5.602e-10 |
| (C) vs explicit model | 1.14e-15 rel | **1.27e-14 rel** (was 6.06e-07) |
| (D) vs numpy NoLoop | 5.821e-11 | 8.222e-10 |

Path B runs at `Omega*T_segment` for a 6-hour signal rather than 90 minutes — needed by (A)
alone, since (B) and (C) are at machine precision across the whole `INFL` range once the
defects are fixed.

The note in this issue that the pre-#142 numbers are not a regression target holds and is
recorded in the test.

All numbers on ldas-pcdev11, CPU, float64. `.travis/test-jax.sh`: 14 passed, skipped=0.
Numpy slowrot suite (11 files): 35 passed.

**Comment by oshaughnessy-junior** [comment]
Fixed and merged in #163 (`bac1c81e`, merged 2026-08-19). Closing by hand: the PR's `Closes #159` keyword did not fire because it targeted `rift_O4d` rather than the repository's default branch.

Outcome, for the record — (A) and (B) are now asserted at `p_max = 0` **and** `p_max = 1`, with no tolerance widened:

|  | p_max=0 | p_max=1 |
|---|---|---|
| (A) static deficit | 4.9865 nats | 3.9234 nats (gate > 1.0) |
| (B) bound deficit | +0.000000e+00 | +5.602e-10 (gate: overshoot <= 1e-6) |
| (C) vs explicit model | 1.14e-15 rel | **1.27e-14** rel (was 6.06e-07) |
| (D) vs numpy NoLoop | 5.821e-11 | 8.222e-10 |

The diagnosis in this issue was wrong and the sweep is what showed it: cutting `fmax` 1700 → 64, a 26x reduction in `2*pi*f*delta_tau`, moved (C) by 0%. The cause was two defects — an odd-`p` Nyquist-bin inconsistency in `time_derivative_weight` (a shared library defect, also carried by the numpy NoLoop) and the test reference's own circular-shift convention. Details in #163.

Twin defect in Path D: #164 / #165 (also merged). Follow-ups still open: #168, #169.
